etc/docker/kernel-image-puller/kernel_image_puller.py

```diff
@@ -4,12 +4,23 @@
 import requests
 import time
 
+from kubernetes import config, client
+
+config.load_incluster_config()
+first_node_name = client.CoreV1Api(client.ApiClient()).list_node().items[0].metadata.name
+node_info = client.CoreV1Api(client.ApiClient()).read_node(name=first_node_name)
+container_runtime = node_info.status.node_info.container_runtime_version
+
+if 'docker://' not in container_runtime:
+    while True:
+        # prevent a daemonset from restarting
+        time.sleep(10)
```

Comment:
Could we log a warning message every 5 minutes or so indicating that KIP isn't doing anything and it's recommended that kernel images be pulled manually prior to their use?  Something like:
```python
logger.warning("KernelInfoPuller only operates on Docker-based K8s systems and is unable to pull kernel images at this time.  Please pre-seed your nodes with the necessary kernel images prior to their use.")
```

The wording for _Docker-based K8s systems_ may not be correct, but that's an idea of what we should let the user/admin know is happening.

Reply:
ok, add the log and sleep longer